### PR TITLE
Adjust server messages on login and map change

### PR DESCRIPTION
**Pull request by Kenpachi2k13**
### Pull Request Prelude

- [x] I have followed [proper Hercules code styling][code].
- [x] I have read and understood the [contribution guidelines][cont] before making this PR.
- [x] I am aware that this PR may be closed if the above-mentioned criteria are not fulfilled.

### Changes Proposed

- Implement `PACKET_ZC_PERSONAL_INFOMATION` (`0x097b`) and use it in `clif_show_modifiers()`.
- Add battle flag for rate modifier messages display.
- Add missing configuration messages `CZ_CONFIG_CALL` and `CZ_CONFIG_PET_AUTOFEEDING` to `clif_parse_LoadEndAck()`.
- Add battle flag for configuration messages display.
- Extend `guild_notice_changemap` battle flag with option for login.
- Add several cases where party options should be sent.
- Add battle flag for sending party options.
- Add battle flag for overweight messages display.
- Do the message sending in new `clif_load_end_ack_sub_messages()` function to have it centralized and be able to control the order.
- Add `clif_open_ui_send()` function to do the actual packet sending for opening an UI.
- Add battle flag for tip of the day window display.

**Issues addressed:** None.

[cont]: https://github.com/HerculesWS/Hercules/blob/master/CONTRIBUTING.md
[code]: https://github.com/HerculesWS/Hercules/wiki/Coding-Style

### CODE REVIEW

**Comment by xBeret** [comment]
Aegis also shows Type Box messages when character first enters.

Use **clif->open_ui(sd, ZC_TIPBOX_UI);**

File: src/map/clif.c [hunk 1]
```diff
@@ -23125,68 +23125,95 @@ static void clif_parse_open_ui_request(int fd, struct map_session_data *sd)
...
-	case CZ_STYLIST_UI:
-		p.UIType = ZC_STYLIST_UI;
+	p.UIType = ui_type;
+
+	switch (ui_type) {
+	case ZC_BANK_UI:
+	case ZC_STYLIST_UI:
+	case ZC_CAPTCHA_UI:
+	case ZC_MACRO_UI:
+	case ZC_TIPBOX_UI:
```
Comment (4144) [inline src/map/clif.c:23153]: will be compilation error for packet versions before 20171122
for fix this, need put some ``ZC_`` cases into ``#ifdef `` blocks

File: src/map/clif.c [hunk 2]
```diff
@@ -23125,68 +23125,95 @@ static void clif_parse_open_ui_request(int fd, struct map_session_data *sd)
...
 
+static void clif_open_ui(struct map_session_data *sd, enum cz_ui_types uiType)
+{
+	nullpo_retv(sd);
+
+	enum zc_ui_types send_ui_type;
+
+	switch (uiType) {
+	case CZ_STYLIST_UI:
+		send_ui_type = ZC_STYLIST_UI;
```
Comment (4144) [inline src/map/clif.c:23201]: here can be compilation errors too for ``ZC_`` constants.

File: src/map/clif.c (hunk 1 above)
Comment (Kenpachi2k13) [reply src/map/clif.c]: Fixed.

File: src/map/clif.c (hunk 2 above)
Comment (Kenpachi2k13) [reply src/map/clif.c:23206]: Fixed.

File: src/map/clif.c [hunk 3]
```diff
@@ -11065,7 +11065,8 @@ static void clif_parse_LoadEndAck(int fd, struct map_session_data *sd)
 		// Notify everyone that this character logged in. [Skotlex]
 		map->foreachpc(clif->friendslist_toggle_sub, sd->status.account_id, sd->status.char_id, 1);
 
-		clif->open_ui_send(sd, ZC_TIPBOX_UI);
+		if (battle_config.show_tip_window != 0)
+			clif->open_ui_send(sd, ZC_TIPBOX_UI);
```
Comment (4144) [inline src/map/clif.c:11069]: error from ci ``clif.c:11069:27: error: use of undeclared identifier 'ZC_TIPBOX_UI'``
this is because old packet version too

Comment (Kenpachi2k13) [reply src/map/clif.c:11070]: Thanks. Fixed, too.

**Pull request merged**